server/client_manager.py

```diff
@@ -175,6 +184,18 @@ def __init__(self, server, transport, user_id, ipid):
             # rainbowtext hell
             self.rainbow = False
 
+            # yt-dlp parameters
+            self.ydl_opts = {
+                    'format': 'ogg/bestaudio/best',
+                    'postprocessors': [{  # Extract audio using ffmpeg
+                        'key': 'FFmpegExtractAudio',
+                        'preferredcodec': 'vorbis',
+                        'preferredquality': '192',
+                    }],
+                    "outtmpl": 'storage/tmp/%(title)s.%(ext)s',
```

Comment:
Why don't you use the Python NamedTemporaryFile object that gives you a file and filename you can use. This way you don't need to worry about this folder having correct perms set by the admin, and also having to clean it up.